### PR TITLE
PREAPPS-4182:Change login options available to the user

### PR DESCRIPTION
**Pull request by madhavdhuppe**
-Implemented functionality to display only classic and modern options on the login page.
-Removed Standard (HTML), Mobile and Touch options from login page
-Removed Webroot/m folder.
-Covered PREAPPS-4210

### CODE REVIEW

File: WebRoot/public/login.jsp [hunk 1]
```diff
@@ -276,51 +270,9 @@
...
-                                        <c:if test="${not fn:contains(ignoredQueryParams, testKey)}">
-                                                <c:param name="${p.key}" value='${value}'/>
-                                            </c:if>
-                                        </c:forEach>
-                                    </c:forEach>
-                                </c:redirect>
-                            </c:when>
-                            <c:when test="${client eq 'zimbrax' and zimbraxSupported}">
+                            <c:when test="${client eq 'modern' and zimbraxSupported}">
                                     <jsp:forward page="/public/zimbrax.jsp"/>
```
Comment (BhavinBathani) [inline WebRoot/public/login.jsp]: Can't this be modern as well ?

File: WebRoot/h/login [hunk 2]
```diff
@@ -2,13 +2,6 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 
 <%-- Pass on all parameters for deeplinking --%>
-<c:url value="search" var="url">
```
Comment (BhavinBathani) [inline WebRoot/h/login]: instead of removing this can't we just remove "/h/login" from web.xml

Comment (madhavdhuppe) [reply WebRoot/h/login]: It's not working that way

File: WebRoot/public/login.jsp (hunk 1 above)
Comment (madhavdhuppe) [reply WebRoot/public/login.jsp]: done.

**Comment by madhavdhuppe** [comment]
> remove all touchclient related code from login.jsp

Initially i thought it was getting used in PREAPPS-4183 i.e reskin login page, Removed

File: WebRoot/h/login (hunk 2 above)
Comment (silentsakky) [reply WebRoot/h/login]: yeah it is very tightly coupled so hard to do now

**Pull request merged**